[ADD] Barcode: New RFID setup docs (#15347)
odoo/documentation: content/applications/inventory_and_mrp/barcode/setup/rfid.rst

====
RFID
====

.. |EPC| replace:: :abbr:`EPC (electronic product code)`

RFID stands for Radio Frequency Identification. RFID enables you to track items using radio waves:
RFID tags are attached to products in inventory, and RFID readers can read those tags to transmit
information to a computer. RFID helps automate processes by keeping inventory accurate and tracking
internal and external moves.

RFID tags transmit electronic product codes (|EPC|), which are unique identifiers to distinguish
individual items. The |EPC| in Odoo is based on GS1 standards to consistently identify products

> worldwide. GS1 nomenclature establishes common rules for encoding information within a barcode to

Reviewer: Would be a great place to link to the GS1 doc! https://www.odoo.com/documentation/19.0/applications/inventory_and_mrp/barcode/operations/gs1_nomenclature.html

facilitate the global exchange of goods.

Hardware
========

> To begin working with RFID in Odoo, you need:

Reviewer: ```suggestion
The following are required to begin working with RFID in Odoo:
```

- Zebra TCX Series mobile computer, or
- Android Smartphone

Paired with:

- Zebra RFD40 (can read and write RFID tags)

> Before you begin

Reviewer: ```suggestion
Configuration
```

================

Before using RFID tags to manage inventory, the APK with RFID support must be installed on the
mobile computer or smartphone.

Before installing the APK on an individual device, you must first enable the ability to install

> unknown apps in the device settings. Find this setting in the device’s :guilabel:`App` or

Reviewer: ```suggestion
unknown apps in the device settings. Find this setting in the device's :guilabel:`App` or
```

:guilabel:`App & Notification` settings, under the :guilabel:`Special access` or :guilabel:`Special

> app access` section.

Reviewer: I would turn this into an admonition block, either important or note.

Then, download the APK. On the :guilabel:`Downloads` page, under the :guilabel:`Mobile Application`
section, click :guilabel:`Download` next to the :guilabel:`Android` option, then select
:guilabel:`Download APK (with Zebra RFID support)` from the drop-down menu. Download this file via a
computer and transfer the APK via USB or Bluetooth, or use the device’s web browser to download the
APK.

.. image:: rfid/download-apk.png
   :alt: Download the APK from the Download page.

Install the APK. If :guilabel:`Install unknown apps` is enabled for the device’s web browser,
install the APK directly from the browser download notification. If :guilabel:`Install unknown apps`
is enabled for the device’s file manager, open it and navigate to the location where the file is
stored, then tap it to install the APK.

Then, when reading or writing RFID tags, ensure that the Zebra device and the smart device are
paired and connected.

Set a compatible barcode
========================

Before you can manage inventory using RFID tags, you must first set a compatible barcode on

> products. Set the product barcode in the product form. To work with RFID, the product barcode must

Reviewer: Would add a short description of how to set the barcode on the product form, or at least link to the doc that explains it

> products. Set the product barcode in the product form. To work with RFID, the product barcode must

Reviewer: ```suggestion
Before inventory can be managed using RFID tags, a compatible barcode must be assigned to the product records. Set the product barcode in the product form. To work with RFID, the product barcode must
```

be GTIN-14-compatible, and inventory tracking must be enabled for the product. Product barcodes that
are not GTIN-14-compatible must be upgraded to work with RFID.

.. note::
    Tracking by serial number or lots does not need to be enabled for this feature to work properly.

Limitations
===========

- The serial number that is part of the |EPC| must only be numeric, as SGTIN-96 is the only

>   supported encoding scheme for RFID tags in Odoo. Alphanumeric serial numbers are not supported.

Reviewer: ```suggestion
  supported encoding scheme for RFID tags in Odoo. Alphanumeric serial numbers are **not** supported.
```

> - For products tracked by lot, you can scan in a lot, but it will be considered as one lot with a

Reviewer: ```suggestion
- For products tracked by lot, the lot can be scanned in, but it will be considered as one lot with a
```
